### DISCUSSION TITLE
multiple plot destinations

### DISCUSSION THREAD

**Discussion by hajes** [madMAx43v3r/chia-gigahorse#91] [General]
is there a way to cycle final destinations, please?

I have twice `-d /c1/ -d /c2/`, but for some unknown reason it writes only to second one.

Idea behind is to fill the slow drive first `/c1` 9-15min/plot copy....meanwhile use the fast `/c2`.

any suggestions, please?

**Comment by madMAx43v3r** [comment]
it prioritizes drives with more free space, if there is not already a copy in progress to that drive.
so once the drives have equal free space it will alternate between them.

**Comment by hajes** [comment]
I tried `plot_sink`, and I noticed it always uses a drive with more space. Once, both drives have same free space...it cycles drives. I like to [KISS](https://eh.hajes.org/KISS), but `plot_sink` at least allows to swap drives on the fly without killing plotter process.
